assets/src/edit-story/app/media/useMediaReducer/reducer.js

```diff
@@ -106,6 +106,16 @@ function reducer(state, { type, payload }) {
       };
     }
 
+    // Why I am getting Cyclomatic Complexity of 21 adding a new statement on this reducer?
```

Comment:
At first glance I count 11 `case` statements and 6 `if` statements with a total of 9 conditions, reaching a total cyclomatic complexity of 20.

You can increase the max allowed complexity for this file using `/* eslint complexity: ["error", { "max": 30 }] */`

However, I'd say at this point it might be reasonable to split out reducers into their own separate files, like in the story reducer.

Reply:
If that break up happens, maybe best in another PR. As not needed for this. Thoughts @merapi

Reply:
Yeah I would do it in a separate PR and for now add this eslint comment here as mentioned